### DISCUSSION TITLE
blog/blog-tech/giscus%EC%A0%81%EC%9A%A9

### DISCUSSION THREAD

**Discussion by giscus** [Samdasoo1076/blog#9] [comment]
# blog/blog-tech/giscus%EC%A0%81%EC%9A%A9

Next.js에 giscus 적용하는법

https://blog-samdasoo.vercel.app/blog/blog-tech/giscus%EC%A0%81%EC%9A%A9

**Comment by Samdasoo1076** [comment]
요롱롱
